pkg/kubectl/cmd/auth/cani.go

```diff
@@ -156,10 +156,10 @@ func (o *CanIOptions) Complete(f cmdutil.Factory, args []string) error {
 func (o *CanIOptions) Validate() error {
 	if o.NonResourceURL != "" {
 		if o.Subresource != "" {
-			return fmt.Errorf("--subresource can not be used with nonResourceURL")
+			return fmt.Errorf("--subresource can not be used with NonResourceURL")
 		}
 		if o.Resource != (schema.GroupVersionResource{}) || o.ResourceName != "" {
-			return fmt.Errorf("nonResourceURL and Resource can not specified together")
+			return fmt.Errorf("NonResourceURL and ResourceName can not specified together")
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
got it. I will revert from NonResourceURL to nonResourceURL

Reply:
@shiywang I think we still should use `NonResourceURL` here. As is mentioned in the link you provided, `NonResourceURL` is the "proper noun"
```
Error strings should not be capitalized (unless beginning with proper nouns or acronyms) or end with punctuation, since they are usually printed following other context.
```

Reply:
all right, you win

Reply:
@shiywang :smile: